omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/PatientIdentifierController1_9Test.java

```diff
@@ -125,6 +126,20 @@ public void shouldVoidAPatientIdentifier() throws Exception {
 	}
 	
 	@Test
+	public void shouldListAllPatientIdentifiersWithVoidedIdentifiersForAPatient() throws Exception {
+		assertEquals(false, service.getPatientIdentifierByUuid(getUuid()).isVoided());
+		MockHttpServletRequest req = request(RequestMethod.DELETE, getURI() + "/" + getUuid());
+		req.addParameter("!purge", "");
+		final String reason = "none";
```

Comment:
If your changes in PatientIdentifierResource1_8.java were removed, would this test fail?

Reply:
Yes, because getAllCount would return 1